README.md

```diff
@@ -162,6 +143,4 @@ I would like to thank Micorverse for the guidances
 
 This project is [MIT](./LICENSE) licensed.
```

Comment:
| 🔴 Broken MIT Link |
| --- |
| ![image](https://github.com/oluwatobi-fasasi/toDoList/assets/93039359/d4fc9f0e-ef6d-41a2-b8c3-135f15eeb06b) |

- [ ] As I previously suggested that it would be nice if you fix your MIT link because it is broken and giving me a 404 error which you can see in the image above 👀 Kindly fix this error to make your readme descriptive and professional 👍 This error is coming because you don't have a LICENSE file in your repo 🤔 Kindly add a LICENSE file using these [guidelines](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to make your project licensed and readme descriptive 😊

Reply:
@ernestkorir, dear, it is broken you can check the MIT link in your readme file